### PR TITLE
update citations

### PR DESCRIPTION
**Pull request by adeepH**
To update citations for [Offenseval_dravidiain](https://huggingface.co/datasets/offenseval_dravidian)

### CODE REVIEW

File: datasets/offenseval_dravidian/README.md [hunk 1]
```diff
@@ -184,6 +184,22 @@ This work is licensed under a [Creative Commons Attribution 4.0 International Li
...
+  Muralidaran, Vigneshwaran and
+  Jose, Navya and
+  Suryawanshi, Shardul and
+  Sherly, Elizabeth  and
+  McCrae, John P",
+  journal={Language Resources and Evaluation},
+  publisher={Springer}
+}
+
+```
```
Comment (lhoestq) [inline datasets/offenseval_dravidian/README.md:202]: This one looks incomplete, is this normal ?

**Review by lhoestq — COMMENTED** [review]
Thanks :) I just have one question regarding the first citation:

File: datasets/offenseval_dravidian/README.md (hunk 1 above)
Comment (adeepH) [reply datasets/offenseval_dravidian/README.md:202]: Hi @lhoestq, thanks for reviewing the PR.
As the manuscript is under review in the Language Resources and Evaluation Journal, which could take months before it gets published online.  Hence, it's a common practice to have a citation like this. I hope this helps.

Comment (lhoestq) [reply datasets/offenseval_dravidian/README.md:202]: makes sense thanks !

**Review by lhoestq — APPROVED** [review]

**Pull request merged**